recipe/bld.bat

```diff
@@ -5,17 +5,18 @@ setlocal EnableDelayedExpansion
 set PKG_CONFIG_PATH="%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig;%BUILD_PREFIX%\Library\lib\pkgconfig"
 set SEARCH_PATH="%BUILD_PREFIX%\Library\"
 
+
 :: meson options
 :: (set pkg_config_path so deps in host env can be found)
-set ^"MESON_OPTIONS=^
-  --prefix="%LIBRARY_PREFIX%" ^
+set MESON_OPTIONS=^
+  --prefix=%LIBRARY_PREFIX% ^
   --wrap-mode=nofallback ^
   --buildtype=release ^
   --backend=ninja ^
+  -Ddoctool=enabled ^
```

Comment:
Does it fail if doctool is disabled?

Reply:
No, it works without it. I enabled it to debug build logs and catch the root cause. MSYS2 enabled it https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-gobject-introspection/PKGBUILD#L54

Reply:
Ok, so I guess we can disable it?

Reply:
Removed